addons/base_import/models/base_import.py

```diff
@@ -450,6 +450,7 @@ def _read_xlsx(self, options):
         sheets = options['sheets'] = book.sheetnames
         sheet_name = options['sheet'] = options.get('sheet') or sheets[0]
         sheet = book[sheet_name]
+        sheet.calculate_dimension()
```

Comment:
so since we don't use the result, there is a magical side effect that will fix the reported issue?

In the code there doesn't seem to be side effect (https://openpyxl.readthedocs.io/en/stable/_modules/openpyxl/worksheet/worksheet.html#Worksheet.calculate_dimension) but I guess if it works there is one.

Reply:
Yea, its here [calculate_dimension](https://foss.heptapod.net/openpyxl/openpyxl/-/blob/branch/default/openpyxl/worksheet/_read_only.py?ref_type=heads#L138) because it is a `ReadOnlyWorksheet`.

Reply:
I updated the code so we only force calculate when the tag is missing.